### PR TITLE
Gabrielalao/fence 1339 upgrade flutter sdk to latest ios and android radar sdk

### PR DESCRIPTION
**Pull request by gabrielalao**
- Bump iOS version from 3.5.9 to 3.8.9
- Bump android version from 3.5.9 to 3.8.12
- remove `setAdIdEnabled`
- rename `sendEvent` to `logConversion` and add `revenue` param
- add `trackVerified`
- add `trackVerifiedToken`
- add `isUsingRemoteTrackingOptions`
- update `autocompleteQuery` with param add `expandUnits`
- add `validateAddress`
- add `App Attest` to ios example
- add `Play Integrity API` to android example
- update example project

### CODE REVIEW

File: android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java [hunk 1]
```diff
@@ -208,9 +198,6 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull final Result result)
                 case "setAnonymousTrackingEnabled":
                     setAnonymousTrackingEnabled(call, result);
                     break;
-                case "setAdIdEnabled":
```
Comment (tjulien) [inline android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java:211]: For breaking changes, we add entries to `MIGRATION.MD`.  This repo doesn't have one yet, but let's create one.  It should be formatted like this: https://github.com/radarlabs/radar-sdk-ios/blob/master/MIGRATION.md

Comment (tjulien) [reply android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java:211]: We can put all of the renamed functions in there as well, like that latest ios MIGRATION files does

File: android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java [hunk 2]
```diff
@@ -1139,6 +1132,100 @@ public void run() {
...
+            @Override
+            public void onComplete(final Radar.RadarStatus status, final RadarAddress address, final Radar.RadarAddressVerificationStatus verificationStatus) {
+                runOnMainThread(new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            JSONObject obj = new JSONObject();
+                            obj.put("status", status.toString());
+                            if (address != null) {
+                                //obj.put("address", address.toJson());
```
Comment (tjulien) [inline android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java:1207]: Let's uncomment this and we'll fix the `NaN` bug in the radar android SDK

File: pubspec.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 name: flutter_radar
 description: Flutter package for Radar, the leading geofencing and location tracking platform
-version: 3.1.7
+version: 3.1.8
```
Comment (tjulien) [inline pubspec.yaml:3]: Let's do `3.2`

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
+# 3.1.8
```
Comment (tjulien) [inline CHANGELOG.md:1]: Let's do `3.2`

File: ios/flutter_radar.podspec [hunk 5]
```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
-  s.dependency 'RadarSDK', '3.5.9'
+  s.dependency 'RadarSDK', '3.8.9'
```
Comment (tjulien) [inline ios/flutter_radar.podspec:13]: above, let's update `s.version` to `3.2`

File: android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java (hunk 1 above)
Comment (gabrielalao) [reply android/src/main/java/io/radar/flutter/RadarFlutterPlugin.java:211]: Yes, @tjulien

**Comment by gabrielalao** [comment]
@tjulien fixed all please review again

**Comment by tjulien** [comment]
Looks good - except `MIGRATION.MD` is empty - can you copy the content from here: https://github.com/radarlabs/radar-sdk-ios/blob/master/MIGRATION.md#36x-to-370. except translate the code to flutter?

**Comment by gabrielalao** [comment]
@tjulien I updated migration.md I thought I pushed the commit.

File: MIGRATION.MD [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Migration guides
+
+## 3.1.x to 3.2.0
+- `Radar.setAdIdEnabled()` has been removed.
+- Custom events have been renamed to conversions.
+    - `Radar.sendEvent(customType, location, metadata)` is now `Radar.logConversion(name, revenue, metadata)`.
+    - The method response does not include `location` and `user` props anymore.
```
Comment (tjulien) [inline MIGRATION.MD:7]: Can you add code snippets below of the migration, for example, how this is done in the native SDKs: https://github.com/radarlabs/radar-sdk-android/blob/master/MIGRATION.md#36x-to-370

Comment (gabrielalao) [reply MIGRATION.MD:7]: Yes. I will

Comment (gabrielalao) [reply MIGRATION.MD:7]: @tjulien added code snippets

**Review by tjulien — APPROVED** [review]

**Pull request merged**